qiskit/visualization/counts_visualization.py

```diff
@@ -61,6 +61,7 @@ def plot_histogram(
     figsize=None,
     color=None,
     number_to_keep=None,
+    ignore_under=None,
```

Comment:
This argument would need to go at the end of the signature, since inserting it in somewhere in the middle can break existing user code that calls this function with positional arguments only. E.g. if someone has
```python
plot_histogram(data, (10, 6), "k", 10, "asc")
```
it now would break.